src/Service/S3/src/Result/CreateMultipartUploadOutput.php

```diff
@@ -125,28 +125,28 @@ public function getSSECustomerAlgorithm(): ?string
     {
         $this->initialize();
 
-        return $this->sSECustomerAlgorithm;
+        return $this->sseCustomerAlgorithm;
     }
 
     public function getSSECustomerKeyMD5(): ?string
     {
         $this->initialize();
 
-        return $this->sSECustomerKeyMD5;
+        return $this->sseCustomerKeyMD5;
     }
 
     public function getSSEKMSEncryptionContext(): ?string
     {
         $this->initialize();
 
-        return $this->sSEKMSEncryptionContext;
+        return $this->sseKmsEncryptionContext;
```

Comment:
Do we care about this?
We use one type of case on the methods, and another on the property.

Reply:
FYI: https://3v4l.org/t5BlU

Reply:
What do you mean?
- just `sseKMS` we just focus on first word only
- rename `getSSEKMS` into `getSseKms` (`getARN` into `getArn`) we apply the same naming rule also on methods

note: renaming method is Backward compatible as you pointed it in your link.

Reply:
I meant that the function is called `getSSEKMSEncryptionContext()` and the property is `sseKmsEncryptionContext`. I like the property name. Should we rename the function too?

Reply:
no strong opinion about this, but it would be ~easy to fix (the method already exists).

By the way, I reverted it, to only camelCase the first word.
 But if we choose to also camelCase `KMS` here I should, for consistency, refactor the function to also fix words that are in the middle of the name (ie `sseCustomerKeyMD5`)

Tell me what you prefer.

Reply:
So I pushed a new commit that also normalize Methods name and class name

This fixes things like:
```php
public function getFOOBar() {
  return $this->fooBar;
}
```

Reply:
hmm.. changed my mind, revert the normalization on class name (this is a BC break because of autoloading)
And keep Method and Property only